AxonDeepSeg/segment.py

```diff
@@ -7,398 +7,403 @@
 
 # Imports
 
-import AxonDeepSeg
-from AxonDeepSeg.apply_model import axon_segmentation
-import os, json, imageio, sys
-from tqdm import tqdm
-import pkg_resources
+import sys
+from pathlib import Path
+import os
+import json
 import argparse
 from argparse import RawTextHelpFormatter
+from tqdm import tqdm
+import pkg_resources
+
+import imageio
+
+import AxonDeepSeg
 import AxonDeepSeg.ads_utils
+from AxonDeepSeg.apply_model import axon_segmentation
 
 # Global variables
 SEM_DEFAULT_MODEL_NAME = "default_SEM_model_v1"
 TEM_DEFAULT_MODEL_NAME = "default_TEM_model_v1"
 
 MODELS_PATH = pkg_resources.resource_filename('AxonDeepSeg', 'models')
+MODELS_PATH = Path(MODELS_PATH)
 
-default_SEM_path = os.path.join(MODELS_PATH,SEM_DEFAULT_MODEL_NAME)
-default_TEM_path = os.path.join(MODELS_PATH,TEM_DEFAULT_MODEL_NAME)
+default_SEM_path = MODELS_PATH / SEM_DEFAULT_MODEL_NAME
+default_TEM_path = MODELS_PATH / TEM_DEFAULT_MODEL_NAME
 default_overlap = 25
 
 # Definition of the functions
 
 def segment_image(path_testing_image, path_model,
-			      overlap_value, config, resolution_model,
-				  acquired_resolution = None, verbosity_level=0):
+                  overlap_value, config, resolution_model,
```

Comment:
Same comment as for "AxonDeepSeg/apply_model.py" re: auto-style formatting, I'll try to parse these two later line-by-line for the differences if the auto-styling can't be easily reversed.